plugins/provenance/server/resource.py

```diff
@@ -329,7 +329,9 @@ def fileSaveHandler(self, event):
                                        level=AccessType.READ, force=True)
         if not item:
             return
-        prevFile = self.model('file').load(curFile['_id'])
+        # should all of these sort of events be forced, since a save has to be
+        # granted first?
+        prevFile = self.model('file').load(curFile['_id'], force=True)
```

Comment:
To answer the question in the comment above this line: we can use `force=True` anytime we want to load the resource without performing an access check for a given user, so really it depends on what we're doing. Your usage here is correct. In some old code a few lines above this is a misleading usage (which should be changed):

```
item = self.model('item').load(id=curFile['itemId'], user=user,
                               level=AccessType.READ, force=True)
```

We should only ever pass a user/level OR force=True, but not both, since force=True simply circumvents access checking altogether.